ch_backup/clickhouse/models.py

```diff
@@ -83,6 +83,64 @@ def _map_paths_to_disks(
             )
         )
 
+    def is_replicated(self) -> bool:
+        """
+        Return True if table engine belongs to replicated merge tree table engine family, or False otherwise.
+        """
+        return self.is_merge_tree() and self.engine.find("Replicated") != -1
+
+    @staticmethod
+    def engine_is_replicated(engine: str) -> bool:
+        """
+        A static method for determining whether an engine is replicated or not.
+        """
+        return Table("", "", engine, [], [], "", "", None).is_replicated()
```

Comment:
Must be adjusted on every change of `Table` constructor.
Why not just `"MergeTree" in engine and "Replicated" in engine` and reuse it in `is_replicated()`